cluster/swarm/cluster.go

```diff
@@ -186,6 +186,14 @@ func (c *Cluster) Image(IdOrName string) *cluster.Image {
 	return nil
 }
 
+func (c *Cluster) Pull(name string, begin, end func(string)) {
+	for _, node := range c.nodes {
```

Comment:
I'll try to find a way, last time I checked, it was messing the docker cli UI completly